.restyled.yaml

```diff
@@ -10,10 +10,6 @@ restylers:
         '--print-width=80',
         '--tab-width=2',
         '--use-tabs=false',
-        '--prose-wrap=always',
+        '--prose-wrap=always'
       ]
-    include:
-      - './*.{js,md}'
-      - 'pages/**/*.js'
-      - 'content/**/*.md'
-      - 'src/**/*.js'
+    include: ['**/*.{js,json,md,yaml,yml}']
```

Comment:
> any reason not to do this?

Yes, it's a white list. For example when you run the dev version there's a .cache/ folder with lots of .js files (also node_modules/ and public/); there can be .vscode/extensions.json in your repo; etc.

Is there a reason to do this? 🙂

Reply:
This is a restyled config file, @jorgeorpinel so that's all uncommitted irrelevance AFAIK.

Reply:
`so that's all uncommitted irrelevance AFAIK.` - your point that it'll be analyzing only changed files?

Reply:
I would double check that this is true

Reply:
the restyled bot works online - it never sees your local mess.